README.md

```diff
@@ -1,2 +1,14 @@
 # saritasa-devops-helm-charts
 Various helm charts
+
+## demo
```

Comment:
Please add good readmes for each our helm tempplate (tekton, tekton-apps, tekton-pipelines). So we would have a documentation with at least parameters which charts use and etc

Reply:
Added, pls review and give the feedback.

Reply:
See README.md in each chart folder.